web/src/beta/lib/reearth-ui/components/TextArea/index.tsx

```diff
@@ -0,0 +1,129 @@
+import { FC, useCallback, useEffect, useState, ChangeEvent, useRef } from "react";
+
+import { fonts, styled } from "@reearth/services/theme";
+
+export type TextAreaProps = {
+  value?: string;
+  placeholder?: string;
+  resizable?: "autoSize" | "default";
+  disabled?: boolean;
+  rows?: number;
+  counter?: boolean;
+  onChange?: (text: string) => void;
+  onBlur?: (text: string) => void;
+};
+
+export const TextArea: FC<TextAreaProps> = ({
+  value,
+  placeholder,
+  resizable,
+  rows,
+  disabled,
+  counter,
+  onChange,
+  onBlur,
+}) => {
+  const [currentValue, setCurrentValue] = useState(value ?? "");
+  const [isFocused, setIsFocused] = useState(false);
+  const textareaRef = useRef<HTMLTextAreaElement>(null);
+
+  useEffect(() => {
+    setCurrentValue(value ?? "");
+    if (resizable === "autoSize" && textareaRef.current) {
+      adjustHeight();
+    }
+  }, [value, resizable]);
+
+  const handleChange = useCallback(
+    (e: ChangeEvent<HTMLTextAreaElement>) => {
+      let newValue = e.currentTarget.value;
+      if (newValue.length > 200 && counter) {
+        newValue = newValue.substring(0, 200);
+      }
```

Comment:
The content length control should be controlled by `maxLength` property (needs to be added to options). This could be handled by HTML textarea naturally so i think we don't need to manage this manually.